### PR TITLE
feat: auth

### PR DESCRIPTION
**Pull request by prakashsakari**
Description:
1. Added Login Form
2. Added Signup Form

Scre
snip5
enshot:
snip6

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
you can use flex and then center the login& signup content otherwise looks ok

File: auth/auth.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    font-size: 1.25rem;
+}
+
+.login-btn {
+    width: 26rem;
+}
+
+.create-account {
+    margin: 1rem 0;
+
```
Comment (Rohitprasad83) [inline auth/auth.css]: Remove unwanted space

File: auth/login.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Rohitprasad83) [inline auth/login.html]: change the title of the page

**Review by Rohitprasad83 — CHANGES_REQUESTED** [review]
import the google fonts to a common CSS file. minor refactoring is needed.

File: auth/auth.css (hunk 1 above)
Comment (prakashsakari) [reply auth/auth.css]: thanks, updated

File: auth/login.html (hunk 2 above)
Comment (prakashsakari) [reply auth/login.html]: thanks, updated

**Pull request merged**